core/commands/refs.go

```diff
@@ -315,8 +322,6 @@ func (rw *RefWriter) WriteEdge(from, to key.Key, linkname string) error {
 		s = strings.Replace(s, "<src>", from.B58String(), -1)
 		s = strings.Replace(s, "<dst>", to.B58String(), -1)
 		s = strings.Replace(s, "<linkname>", linkname, -1)
-	case rw.PrintEdge:
-		s = from.B58String() + " -> " + to.B58String()
 	default:
 		s += to.B58String()
```

Comment:
Do we still need this default case? It would only trigger if `--format` is explicitly set to the empty string, which seems like a corner case better handled by either an error, or actually printing an empty string for each edge, rather than overriding the explicit intention of the user.